packages/sources/coinmetrics/src/transport/price-ws.ts

```diff
@@ -96,12 +90,16 @@ export const handleAssetMetricsMessage = (
     // Is WsAssetMetricsErrorResponse
     logger.error(message, `Error response from websocket`)
 
-    const regex = /'([^']+)'/g
+    const findBaseCurrenciesRegex = new RegExp(/'([^']+)'/g)
     if (message['error']['type'] === 'bad_parameters') {
-      const matches = [...message['error']['message'].matchAll(regex)]
+      //Bad Parameter Error Message
+      //   {error: {
+      //     type: "bad_parameter",
+      //     message: "Metric 'ReferenceRateBTC' with frequency '1s' is not supported for 'cron'."}}
+      const matches = [...message['error']['message'].matchAll(findBaseCurrenciesRegex)]
```

Comment:
Minor: Can just use the `.` syntax which is typically more commonly used in TypeScript, rather than the `[]`

```suggestion
      const matches = [...message.error.message.matchAll(findBaseCurrenciesRegex)]
```

Reply:
noted.